### DISCUSSION TITLE
How do create the windows binary for a project

### DISCUSSION THREAD

**Discussion by JayTea144** [community/community#43609] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,

I don't know where else to ask this, there is no way to leave any question on the project page or any forum.
But how I can get or make the Windows binary (.exe) file for this project - "podsync-with-yt-dlp"?

Thanks.

**Comment by jfullstackdev** [comment, marked as answer]
you can raise an issue at the parent repo, for sure, others have an idea 

https://github.com/mxpv/podsync

**Reply by JayTea144** [reply]
Thank you very much.
